uk.co.compucorp.civicrm.hrleaveandabsences/views/shared/directives/leave-request-popup.html

```diff
@@ -214,40 +214,38 @@ <h4 class="modal-title">
             </uib-tab>
             <uib-tab heading="Files">
               <div>
-                <ul class="chr_leave-request-files_list container-fluid">
-                  <li ng-repeat="file in $ctrl.request.files">
-                    <span ng-if="!file.toBeDeleted">
-                      <div class="chr_leave-request-files_meta">
-                        <span class="chr_leave-request-files_author">{{$ctrl.getCommentorName(file.contact_id)}}</span>
-                        <span>-</span>
-                        <span class="chr_leave-request-files_date">{{$ctrl.formatDateTime(file.upload_date)}}</span>
+                <ul class="chr_leave-request-files_list container-fluid" ng-if="$ctrl.canShowFileList()">
+                  <li ng-repeat="file in $ctrl.request.files | filter: {toBeDeleted: '!'}">{{file.toBeDeleted}}
+                    <div class="chr_leave-request-files_meta">
+                      <span class="chr_leave-request-files_author">{{$ctrl.getCommentorName(file.contact_id)}}</span>
+                      <span>-</span>
+                      <span class="chr_leave-request-files_date">{{$ctrl.formatDateTime(file.upload_date)}}</span>
+                    </div>
+                    <div class="row">
+                      <div class="chr_leave-request-files_icon text-primary col-xs-8">
+                        <i
+                          class="fa fa-file-o"
```

Comment:
Why do we need `fa fa-file-o` when we are already mentioning it inside `ng-class`?